docs/BUILDER_INTEGRATION.md

```diff
@@ -317,6 +317,29 @@ You then have several options:
 
   By using an existing BLOCK_TYPE already used by a flow-spec block, you can override the default store for that flow spec block (e.g. to change how to validate your extended customised version)
 
+### Customizing block output branching explanatory texts
+
+Every `blockClass` may have one or more explanatory texts for various exits
+configurations
+
+```
+"explanatory_texts": [
+    {
+        "title": "flow-builder.multiple-choice-mode",
+        "description": "flow-builder.explanation-mcq-mc",
+        "show_for_branching_types": [
+            "EXIT_PER_CHOICE"
+        ]
+    }
+]
+```
+
+`title` and `description` are displayed as a list of definitions under the
+Output Branching config of block's sidebar editor.
+
+`show_for_branching_types` regulates which output branching settings should
+should make a particular text visible; all tyoes are acceoted by default.
```

Comment:
minor: typos, can be replaced with `'show_for_branching_types' regulates which output branching settings should make a particular text visible; all types are accepted by default.` if there is time